SmartCar/Arduino/sketch_main/sketch_main.ino

```diff
@@ -87,25 +176,29 @@ void loop() {
         completeCommand = false;
     }
 
```

Comment:
Add a few line breaks here to seperate flame stuff from other stuff. So it's easier to locate it.